src/main/java/io/orten/nano/impl/UserService.java

```diff
@@ -0,0 +1,70 @@
+package io.orten.nano.impl;
+
+import io.orten.nano.model.User;
+import io.orten.nano.util.Database;
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserService {
+
+        public static List<User> u_list = new ArrayList<User>();
+
+        public static User getUser(long userID) throws Exception {
+            Session session = Database.getSession();
+            User user = session.get(User.class, userID);

```

Comment:
If this throws an exception then the session isn't closed and you have a resource leak.
This should probably be in a try-finally block

Reply:
I have fixed it